scripts/jenkins/ardana/ansible/roles/input_model_generator/templates/input_model/data/network_groups.yml

```diff
@@ -24,7 +24,19 @@
 
     - name: {{ network_group.name|upper }}
       hostname-suffix: {{ network_group['hostname_suffix'] }}
-{%   if 'MANAGEMENT' in network_group['component_endpoints'] %}
+{%   if ('MANAGEMENT' and 'IPV4')  in network_group['component_endpoints'] %}
```

Comment:
If I understand your intent, then you want to check whether both `MANAGEMENT` and `IPv4` are in the `component_endpoints` network group. Are you sure Jinja understands the way you wrote this? A quick google didn't come up with anything on whether this syntax is supported. (This may just be due to my inadequate search skills :wink:)

Reply:
I tested it and it works as it seems. 
but more testing will brings any other problems software.

Reply:
I see this syntax in other places as well, which hopefully means that it's fine. I just struck me as unusual.

Reply:
I tried this:
```
In [1]: import jinja2
In [2]: template = jinja2.Template("{% if ('a' and 'b') in ['a', 'b', 'c'] %} yes {% else %} no {% endif %}")
In [3]: template.render()
Out[3]: ' yes '
```
So it looks like it's valid syntax.